shared/src/main/scala/Parser.scala

```diff
@@ -335,7 +335,10 @@ object Parser:
 
   private def parseParameters(params: AST): (List[String | Int], Int) =
     val paramString = params.toVyxal
-    val components = paramString.split(",")
+    val components = paramString.split(
+      ","
+    ) // may leave extra spaces, but that's okay, because
+    // spaces are removed when converting to a valid name
```

Comment:
```suggestion
    val components = paramString.split(",")
    // ^ may leave extra spaces, but that's okay, because
    // spaces are removed when converting to a valid name
```
Just wanted to get that `split(",")` all on one line